reference_designs/ipu-xrt/matrix_multiplication_column/Makefile

```diff
@@ -0,0 +1,69 @@
+##===- Makefile -----------------------------------------------------------===##
+# 
+# This file licensed under the Apache License v2.0 with LLVM Exceptions.
+# See https://llvm.org/LICENSE.txt for license information.
+# SPDX-License-Identifier: Apache-2.0 WITH LLVM-exception
+# 
+##===----------------------------------------------------------------------===##
+
+include ../makefile-common
+
+M?=128
+K?=128
+N?=128
+
+m?=64
+k?=32
+n?=64
+
+dtype_in?=i16
+ifeq ($(dtype_in), i16)
+	word_size_in?=2
+	r?=4
+	s?=4
+	t?=4
+	dtype_out?=i16
+	word_size_out?=${word_size_in}
+endif
+ifeq ($(dtype_in), bf16)
+	word_size_in?=2
+	r?=4
+	s?=8
+	t?=4
+	dtype_out?=f32
+	word_size_out?=4
+endif
```

Comment:
I believe lines 11 to 35 could be removed to avoid confusion as I don't see these environment variables used anywhere anymore. Unless you plan to adjust the Python script to take them as input arguments again and want to keep those defaults, which would be cool.